### ISSUE TITLE
[UNBAN] Making PokeBuddy unban exploit via v.0.33.0 - Coming soon

### ISSUE THREAD

**Issue by xReVoLT-iTheme** [finndev/PokeBuddy#1237]
After hours of research, I recently found that almost every PokeBuddy user has gotten their account banned from the developers of Pokemon GO known as 'Niantic', I being myself, love to twist and rebel against these rules and unban myself and others. Why not fight back, right? That's when I come in.

Over the past few days I've been modding Pokemon GO in various ways from in-app purchase exploits(still working on it), to modding pokemon, to even adding your own pokemon. Once the trade feature has been added, I'll be able to make the exploit of making your own pokemon.

Now, for some proof, here's me doing some address lookup via pokemon-go-v0330.apk of modding in app purchases(still need to modify some values and inject my own code):

[](http://i.imgur.com/UrCIMW6.png)

Also, I can modify my character:

[](http://i.imgur.com/2UOF6gh.png)
I can modify pokemon:

[](http://i.imgur.com/kxzy343.png)

I can modify attacks:

[](http://i.imgur.com/kxzy343.png)

Even the modify the environment(local only):

[](http://i.imgur.com/7hBfFgq.png)

And a lot more...!

But, what really makes me interested is that recently within the week I believe, many users have experienced themselves being permanently banned with PokeBuddy. Due to this, I thought I might as well 'experiment' and try to unban myself and if successful, others. Wish me the best of luck and stay tuned(come back daily). We will live on!

**Comment by ak1rau** [comment]
GO GO GO!!!!

**Comment by xXbrucyjucyXx** [comment]
Dude......lmk i want my lvl 32 back....

**Comment by Enviouse** [comment]
Please keep me in the loop nitanic fucked us all lets fuck them I would like my lv30 acc back if u figure the un ban out let me no they failed to tell me anything not even a email

**Comment by xXbrucyjucyXx** [comment]
Yeah they try to goat you into messaging them about the ban.....even if you
message them about a faulty login you just get auntomated messages

**Comment by Enviouse** [comment]
I didn't want to msg them but afta a week I did got that auto shit I wonder if anyone has un banned the acc yet

**Comment by malmo0** [comment]
Good luck man, i was on level 39 😫

**Comment by Yocylia** [comment]
!!!

**Comment by ulkem93** [comment]
Thanks for the efforts bro

**Comment by xXbrucyjucyXx** [comment]
Yesh thats what happened to me to its automated if you didnt recieve email
origi ally you shouldnt have put in a request and brought that much
attention to yourself

On Aug 20, 2016 2:36 AM, "Enviouse" <notifications@github.com> wrote:

I didn't want to msg them but afta a week I did got that auto shit I wonder
if anyone has un banned the acc yet

—
You are receiving this because you commented.
Reply to this email directly, view it on GitHub
<https://github.com/finndev/PokeBuddy/issues/1237#issuecomment-241183078>,
or mute the thread

**Comment by futi666** [comment]
i nee my lvl 33 acc back :( go go go

**Comment by Yocylia** [comment]
i think it wouldn't work..niantic will find out and its in their system which account names and its id is already ban..they will just ban back

**Comment by xReVoLT-iTheme** [comment]
Thread updated. Proof is now added(unfortunately the link tag erased the picture, so I had to add it once more).

Please stay tuned!

**Comment by malmo0** [comment]
Good work man, hope you make it

**Comment by ulkem93** [comment]
Keep up the good work bro

**Comment by xReVoLT-iTheme** [comment]
Updated - Added href image.

**Comment by Ghekkoo** [comment]
Even if you dont succeed.. just know that if you get banned any new character you make and cheat with is still cooler than any account that doesn't cheat... but please keep messing with pokemon go lmfao

**Comment by matt2310** [comment]
YES LET US KNOW I NEED MY 2 ACCOUNTS BACK!!!!

**Comment by Yocylia** [comment]
GO ANONYMOUS!  FKING KILL DA ILLUMINATI!!!

**Comment by CagasAgua** [comment]
Fighting the good fight

**Comment by NegroSorete** [comment]
I want my main ACC back :(

**Comment by Freerider007** [comment]
Great work! I would like to get my account back as well. They banned me for botting, so I actually started botting now... Cant let it go just like that...

**Comment by WolfKetchum** [comment]
GO MAN! YOU ROCK MY BRO! And please don't make it a paid app or something.

**Comment by ulkem93** [comment]
@xReVoLT-iTheme is the new jesus

**Comment by matt2310** [comment]
Wait can you unban people now?

**Comment by lukebiggz21** [comment]
Lost my lvl 25 account about an hour ago and I haven't even cheated, ī hope you find a way buddy

**Comment by Enviouse** [comment]
Can just ask how u get banned my main was needed over a week ago so I am botting new 1 still no ban lv 23 I keep it local n kinky hit the same 10 stops Pokemon about 1 every 3 mins so not taking advantage botting safe n no ban yet

**Comment by raffa5312** [comment]
hey you figured it out?

**Comment by xXbrucyjucyXx** [comment]
Js they are going hard at botters second account tanked again

**Comment by WolfKetchum** [comment]
Now it's 0.50.0

**Comment by crilleaz** [comment]
This sounds very weird, if you can unban accounts that will be big.

**Comment by Yocylia** [comment]
don think this gonna happen... r . i p .  T_T

**Comment by xXbrucyjucyXx** [comment]
Yeah gonna have to agree the ban is on there side not ours its not like you have your info just locked inside the app on your phone its in there servers.......this would take abhuge exploit to make happen not just a 3rd party program....that is literally how they are flagging people 3rd party clients and programs.......the only one iv seen thats still working is thebapple one thats like a modded app

**Comment by xReVoLT-iTheme** [comment]
@xXbrucyjucyXx Have you ever seen server sided exploits, such as... exploiting on ROBLOX or exploiting on CoD, or any online game really? Non server sided exploits is a process called 'binary exploitation', but I need to use this process to fetch the data which is binary to connect to the servers with my own custom bootup scripts which will run with Pokemon GO. Simple. As of now, in the new updates they have changed security. Therefore, I must start over on this exploit.

**Comment by Yocylia** [comment]
may u provide us with ur current progress if its going to be completed like before the latest update, u were close to like 90% completion but u have to re-do the whole thing as of the new update....

so right now, whats the ~%? completion of ur exploit???

**Comment by ulkem93** [comment]
Well our dreams probably won't come true since its been a long time and no solution so gg everyone

**Comment by Yocylia** [comment]
gg...wp...niantic wins....Annoymous lose!... the end is nigh!

**Comment by xReVoLT-iTheme** [comment]
Right now I'm making the website for the exploit using Android Material theme. So far, it's looking quite unique. Once completed(probably next week), it shall be released.

**Comment by Yocylia** [comment]
pls if this is finally released, gotta show us a step by step tutorial !!!

**Comment by Enviouse** [comment]
Unfortunately it may be a week to late nitanic have already starting to del Accts so ur info may no longer exist great work tho will try to see if it works

**Comment by Yocylia** [comment]
well hack into their system and just inject a lvl 40? i think its possible = )

**Comment by Enviouse** [comment]
Hacking there system will lead them to u so no making a exploit n adding it to there servers good idea

**Comment by Yocylia** [comment]
7 days passed...no exploit yet...rip banned accounts....the dream is just an illusion? T_T

**Comment by Enviouse** [comment]
The new buddy updat will be here this week that's y the exploit isn't done coz if he gives it now he will be updating it by end of week
